server/src/main/java/org/elasticsearch/action/admin/cluster/node/reload/NodesReloadSecureSettingsRequest.java

```diff
@@ -17,62 +17,69 @@
  * under the License.
  */
 
-package org.elasticsearch.action.admin.cluster.reinit;
+package org.elasticsearch.action.admin.cluster.node.reload;
 
 import org.elasticsearch.action.ActionRequestValidationException;
 import org.elasticsearch.action.support.nodes.BaseNodesRequest;
 import org.elasticsearch.common.io.stream.StreamInput;
 import org.elasticsearch.common.io.stream.StreamOutput;
-
 import java.io.IOException;
 
 import static org.elasticsearch.action.ValidateActions.addValidationError;
 
 /**
- * Request for an update cluster settings action
+ * Request for a reload secure settings action
  */
-public class NodesReInitRequest extends BaseNodesRequest<NodesReInitRequest> {
+public class NodesReloadSecureSettingsRequest extends BaseNodesRequest<NodesReloadSecureSettingsRequest> {
 
-    private String secureStorePassword;
+    /**
+     * The password which is broadcasted to all nodes, but is never stored on
+     * persistent storage. The password is used to reread and decrypt the contents
+     * of the node's keystore (backing the implementation of
+     * {@code SecureSettings}).
+     */
+    private String secureSettingsPassword;
```

Comment:
why don't we use a SecureString here? I know this is not anything new in this PR so we can handle in a follow up.